app/src/main/java/com/yeonae/chamelezone/view/search/adapter/SearchTabAdapter.kt

```diff
@@ -3,20 +3,20 @@ package com.yeonae.chamelezone.view.search.adapter
 import androidx.fragment.app.Fragment
 import androidx.fragment.app.FragmentManager
 import androidx.fragment.app.FragmentPagerAdapter
-import com.yeonae.chamelezone.view.search.AreaNameTabFragment
+import com.yeonae.chamelezone.view.search.AddressTabFragment
 import com.yeonae.chamelezone.view.search.KeywordTabFragment
 import com.yeonae.chamelezone.view.search.PlaceNameTabFragment
 
 class SearchTabAdapter(fm: FragmentManager) : FragmentPagerAdapter(fm) {
 
     private val placeNameTabFragment = PlaceNameTabFragment()
-    private val areaNameTabFragment = AreaNameTabFragment()
+    private val addressTabFragment = AddressTabFragment()
     private val keywordTabFragment = KeywordTabFragment()
 
     override fun getItem(position: Int): Fragment {
         return when (position) {
             0 -> placeNameTabFragment
```

Comment:
```suggestion
            0 -> PlaceNameTabFragment()
```
이런식으로 사용하는 것이 좋겠어요

fragment 객체에 접근이 필요하다면 fragmentManager에서 tag로 찾아서 접근하라고 소개되어 있더라고요

Reply:
수정했습니당